examples/python/printing-client/printing-client.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2018 Infostellar, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The Python implementation of the gRPC stellarstation-api client."""
+
+import base64
+import os
+import time
+
+import grpc
+import stellarstation_pb2
+import stellarstation_pb2_grpc
+
+from google import auth as google_auth
+from google.auth import jwt as google_auth_jwt
+from google.auth.transport import grpc as google_auth_transport_grpc
+
+os.environ['GRPC_SSL_CIPHER_SUITES'] = 'ECDHE-RSA-AES128-GCM-SHA256'
+
+SATELLITE_ID = '5'
+
+
+def run():
+  # Load the private key downloaded from the StellarStation Console.
+  credentials = google_auth_jwt.Credentials.from_service_account_file(
+      '../../fakeserver/src/misc/api-key.json',
+      audience='https://api.stellarstation.com')
+
+  # Setup the gRPC client.
+  jwt_creds = google_auth_jwt.OnDemandCredentials.from_signing_credentials(
+      credentials)
+  channel_credential = grpc.ssl_channel_credentials(
+      open('../../fakeserver/src/main/resources/tls.crt', 'br').read())
+  channel = google_auth_transport_grpc.secure_authorized_channel(
+      jwt_creds, None, 'localhost:8080', channel_credential)
+  client = stellarstation_pb2_grpc.StellarStationServiceStub(channel)
+
+  # Open satellite stream
+  request_iterator = generate_request()
+  for value in client.OpenSatelliteStream(request_iterator):
+    print(
+        "Got response: ",
+        base64.b64encode(value.receive_telemetry_response.telemetry.data)[:100])
+
+
+# Since this generator is used as an argument of OpenSatelliteStream,
```

Comment:
This generator yields the requests to send on the stream opened by `OpenSatelliteStream`. The client side of the stream will be closed when this generator returns (in this example, it never returns).